### PR TITLE
Kills recent codersprites

### PR DESCRIPTION
**Pull request by alsoandanswer**
- imageadd: "New sprites for improvised swords, Pesti-shocks, and Koko bars." and power drills
  - tweak: "Fixes Koko related grammar."

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm [hunk 1]
```diff
@@ -1211,7 +1211,7 @@
 /datum/reagent/mental/kokoreed
 	name = "Koko Reed Juice"
 	id = "kokoreed"
-	description = "Juice from the Koko reed plant. Caused unique mental effects in unathi"
+	description = "Juice from the Koko reed plant. Causes unique mental effects in Unathi"
```
Comment (MarinaGryphon) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm]:
```suggestion
	description = "Juice from the Koko reed plant. Causes unique mental effects in Unathi."
```

Comment (alsoandanswer) [reply code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm]: woops lol

**Comment by alsoandanswer** [comment]
!review

**Pull request merged**